ci-operator/step-registry/baremetal/lab/ipi/virtual-media/baremetal-lab-ipi-virtual-media-workflow.yaml

```
@@ -3,6 +3,7 @@ workflow:
   steps:
     pre:
       - chain: baremetal-lab-ipi-conf-virtual-media
+      - ref: baremetal-lab-ipi-conf-network-vips
```

The following piece of code can be removed from `baremetal-lab-ipi-install-commands.sh`, and the `baremetal-lab-upi-conf-network` step can be added here.
```
networking:
  machineNetwork:
  - cidr: ${INTERNAL_NET_CIDR}
 ```
```suggestion
      - ref: baremetal-lab-ipi-conf-network-vips
      - ref: baremetal-lab-upi-conf-network
```